src/wasm-lib/kcl/src/std/extrude.rs

```diff
@@ -181,6 +181,48 @@ pub(crate) async fn do_post_extrude(
         vec![]
     };
 
+    for face_info in face_infos.iter() {
+        if face_info.cap == kittycad::types::ExtrusionFaceCapType::None
+            && face_info.face_id.is_some()
+            && face_info.curve_id.is_some()
+        {
+            match args
+                .batch_modeling_cmd(
+                    uuid::Uuid::new_v4(),
+                    kittycad::types::ModelingCmd::Solid3DGetOppositeEdge {
+                        edge_id: face_info.curve_id.unwrap(),
+                        object_id: sketch_group.id,
+                        face_id: face_info.face_id.unwrap_or_default(),
+                    },
+                )
+                .await
+            {
+                Ok(info) => info,
+                Err(e) => {
+                    eprintln!("Error fetching opposite edge: {:?}", e);
+                    continue;
+                }
```

Comment:
Are these `Err` necessary? Do the errors from batch_modeling_cmd not already give this information?

Reply:
This pattern seems bad to me. In WebAssembly, `eprintln!()` is a no-op. So we're swallowing the error. We can't propagate it?

Reply:
I'm with jon I'd like to see this propagated like all the other errors

Reply:
how's that?